plugins/default/src/Ide/Plugin/Eval/Tutorial.hs

```diff
@@ -0,0 +1,231 @@
+module Ide.Plugin.Eval.Tutorial where
```

Comment:
Wouldn't it be useful to use Haddock markup consistently in this module, so that it is rendered nicely?

Reply:
The Tutorial module was actually meant to be read as source code (so that one can play with it while reading and can see exactly how tests are written down in different kinds of comments).

But you are right, it also makes sense to make it into a proper Haddock-ready module so that it can work as a static README. I modified it accordingly.

Are haddocks built and accessible on the project GitHub site?